schedule.py

```diff
@@ -0,0 +1,42 @@
+"""Module for realtime schedule management."""
+
+import asyncio
+import time
+from typing import Any
+
+from tracker import Tracker
+
+Schedule = dict[int, Any]
+
+
+class ScheduleTracker(Tracker[Schedule]):
+    """Holds schedule in real time.
+    
+    Attributes:
+        modified: The event set when any new modification to the schedule is added.
+        latest: The timestamp of the last modification.
+    """
+
+    def __init__(self):
+        """Extended."""
+        super().__init__()
+        self.modifed = asyncio.Event()
+        self.latest = -1
+
+    def get(self) -> tuple[float, Schedule]:
+        """Returns the current timestamp and schedule."""
```

Comment:
"the current timestamp" looks awkward since it is actually the timestamp of the last modification.
If you don't want to repeat the definition of `latest`, you may write e.g. `Returns the latest timestamp and the current schedule`.